composer.json

```diff
@@ -12,7 +12,8 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "~1.0@dev",
+        "php-http/httplug": "dev-master",
```

Comment:
no, for now we should not change our code in that way. but with
^1.0.0-alpha2 as minimum requirement, the lowest version build and
stable build should still always pick master atm.